src/sentry/runner/commands/configoptions.py

```diff
@@ -30,7 +30,19 @@ def _attempt_update(key: str, value: Any, drifted_options: Set[str], dry_run: bo
         # channel. Thus, if the laast update channel was already
         # UpdateChannel.AUTOMATOR, and the value we are trying to set
         # is the same as the value already stored we do nothing.
-        if options.get_last_update_channel(key) != options.UpdateChannel.AUTOMATOR:
+        last_update_channel = options.get_last_update_channel(key)
+        if last_update_channel is None:
+            # Here we are trying to set an option with a value that
+            # is equal to its default. There are valid cases for this
+            # behavior: I plan to change the default value of an option
+            # without changing the production behavior. So I would
+            # first set the option to the current default value in
+            # the DB and then change the default value.
+            if not dry_run:
```

Comment:
Can you give me an example of how this happens? How does the last-update-channel become None and does changing the default involve changing the option on multiple channels (changing in the Db)?

Reply:
last-update-channel is None if the option is not set in the DB.